### PR TITLE
DOC: Fixed warning about convert_objects in doc build

### PR DESCRIPTION
**Pull request by nadebu**
`convert_objects` doesn't exist anymore, and the example in the release notes when it was added is failing and generating a warning in the doc build:
```
Exception in /home/travis/build/pandas-dev/pandas/doc/source/whatsnew/v0.11.0.rst at block ending on line None
Specify :okexcept: as an option in the ipython:: block to suppress this message
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-20-d11827bcdf31> in <module>
----> 1 s.convert_objects(convert_dates='coerce')
~/build/pandas-dev/pandas/pandas/core/generic.py in __getattr__(self, name)
   5095             if self._info_axis._can_hold_identifiers_and_holds_name(name):
   5096                 return self[name]
-> 5097             return object.__getattribute__(self, name)
   5098 
   5099     def __setattr__(self, name, value):
AttributeError: 'Series' object has no attribute 'convert_objects'
```

Open during the London sprint.

- [X] xref #24173
- [ ] tests added / passed
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

**Comment by WillAyd** [comment]
Thanks for the PR! What warning were you getting? Any easy way to detect others?

**Comment by datapythonista** [comment]
I think all the CI failures are unrelated, not sure what's broken there.

But looks like the warning is still in the docs. @nadebu can you see if you find what's wrong?

@WillAyd added more info to the description.

**Review by datapythonista — CHANGES_REQUESTED** [review]
@nadebu I see that one file (`doc/esample.feather` was added by mistake). Can you please remove it?

Also, I think your change is probably fixing one of the errors, but a similar error is happening in the example immediately after in the PR. Can you apply the same change there please?

Thanks!

File: doc/source/whatsnew/v0.11.0.rst [hunk 1]
```diff
@@ -106,7 +106,7 @@ Conversion
 Mixed Conversion
```
Comment (jreback) [inline doc/source/whatsnew/v0.11.0.rst]: can you remove the added doc/example.feather file?

File: doc/source/whatsnew/v0.11.0.rst [hunk 2]
```diff
@@ -106,7 +106,7 @@ Conversion
...
 
 .. ipython:: python
-   :okwarning:
```
Comment (jreback) [inline doc/source/whatsnew/v0.11.0.rst]: rather than this can you change to a code-block (and just show what it was)

**Review by jreback — CHANGES_REQUESTED** [review]

**Comment by jorisvandenbossche** [comment]
@nadebu sorry, it have been working the last days on the documentation, and this has already been fixed in the meantime.

**Comment by jorisvandenbossche** [comment]
Hopefully you can find another issue to work on! (there are some labeled with good-first-issue)

**Comment by jorisvandenbossche** [comment]
Also, specifically for https://github.com/pandas-dev/pandas/issues/24173 (flake8 in the docs), there are still files left to do, so very welcome to open PRs for those.

**Comment by nadebu** [comment]
Sorry I missed all this conversation. My mail was being diverted and still familiarising myself with github's messaging.

**Pull request closed without merging**